core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -299,6 +299,16 @@ public String getUfsFingerprint() {
     return mInfo.getXAttr();
   }
 
+  /**
+   * This is an experimental API. The returned {@link FileInfo} object does not have a stable API.
+   * Make modifications to the returned file info object at your own risk.
+   *
+   * @return the underlying file info object
+   */
+  public FileInfo getFileInfo() {
+    return mInfo;
+  }
+
```

Comment:
Is there a `@Experimental` annotation or something similar we can use?

Reply:
I tried to look for it, but I think we've done away with that annotation